examples/cdk/package.json

```diff
@@ -18,7 +18,7 @@
     "package-bundle": "echo 'Not applicable'",
     "test:unit": "npm run build && jest",
     "test:e2e": "echo 'To be implemented ...'",
-    "version": "npm install @aws-lambda-powertools/logger@0.11.0 @aws-lambda-powertools/tracer@0.11.0 @aws-lambda-powertools/metrics@0.11.0 && git add package.json",
+    "version": "npm install @aws-lambda-powertools/logger@1.0.1 @aws-lambda-powertools/tracer@1.0.1 @aws-lambda-powertools/metrics@1.0.1 && git add package.json",
```

Comment:
Suggestion: by doing this here and in the SAM example we don't need to do manual amendments anymore and you don't need to change the versions in the dependencies object. 

```suggestion
    "version": "npm install @aws-lambda-powertools/logger@latest @aws-lambda-powertools/tracer@latest @aws-lambda-powertools/metrics@latest && git add package.json",
```
I tested this in a local outdate/old branch: 
<img width="1437" alt="Screenshot 2022-07-14 at 22 07 00" src="https://user-images.githubusercontent.com/47529391/179074015-c06aeae2-8ac2-4dbf-871f-0ef09c941e50.png">